### ISSUE TITLE
[ARES Remediate] Low color contrast in Navigation sidebar

### ISSUE THREAD

**Issue by rladdusaw** [pulibrary/dacs_handbook#336]
### Steps to recreate
1. Login to ARES.
2. Select the "Library Sandbox" Course.
3. Select the option "Reserves" from the course menu.

### URL
[Canvas](https://canvas.princeton.edu/)

### Issues
Element has insufficient color contrast

### Severity
High

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Text of selected section of the application could be difficult to read

### Recommended fix
Increase contrast to at least 4.5:1

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
Current contrast is 3.72:1
